apps/wallet/configs/ts/tsconfig.common.json

```diff
@@ -45,6 +45,7 @@
             "_font-icons/*": ["./font-icons/*"],
             "@mysten/sui.js": ["../../sdk/typescript/src/"],
             "@mysten/bcs": ["../../sdk/bcs/src/"],
+            "@mysten/ledgerjs-hw-app-sui": ["../../sdk/ledgerjs-hw-app-sui"],
```

Comment:
Added this as a `path` instead of a `reference` since that seems to be the pattern in the wallet